projects/1.21/assets/expanded-delight/expandeddelight/lang/zh_cn.json

```
@@ -0,0 +1,144 @@
+  "item.expandeddelight.cinnamon_rice": "肉桂米饭",
+  "item.expandeddelight.cinnamon_apples": "肉桂苹果",
+  "item.expandeddelight.chocolate_cookie": "巧克力曲奇",
+  "item.expandeddelight.sugar_cookie": "糖曲奇",
+  "item.expandeddelight.snickerdoodle": "肉桂脆饼",
+
+  "item.expandeddelight.apple_juice": "苹果汁",
+  "item.expandeddelight.sweet_berry_juice": "甜浆果汁",
+  "item.expandeddelight.glow_berry_juice": "荧光浆果汁",
+  "item.expandeddelight.cranberry_juice": "蔓越莓汁",
+
+  "item.expandeddelight.sweet_berry_jelly": "甜浆果果冻",
+  "item.expandeddelight.glow_berry_jelly": "荧光浆果果冻",
+  "item.expandeddelight.cranberry_jelly": "蔓越莓果冻",
+
+  "item.expandeddelight.cinnamon_boat": "肉桂木船",
+  "item.expandeddelight.cinnamon_chest_boat": "肉桂木箱船",
+
+  "block.expandeddelight.cinnamon_sapling": "肉桂树苗",
+  "block.expandeddelight.cinnamon_leaves": "肉桂树叶",
+  "block.expandeddelight.cinnamon_log": "肉桂原木",
+  "block.expandeddelight.stripped_cinnamon_log": "去皮肉桂原木",
+  "block.expandeddelight.cinnamon_wood": "肉桂木",
+  "block.expandeddelight.stripped_cinnamon_wood": "去皮肉桂木",
+  "block.expandeddelight.cinnamon_planks": "肉桂木板",
+  "block.expandeddelight.cinnamon_stairs": "肉桂楼梯",
+  "block.expandeddelight.cinnamon_slab": "肉桂台阶",
+  "block.expandeddelight.cinnamon_fence": "肉桂栅栏",
+  "block.expandeddelight.cinnamon_fence_gate": "肉桂栅栏门",
+  "block.expandeddelight.cinnamon_door": "肉桂门",
+  "block.expandeddelight.cinnamon_trapdoor": "肉桂活板门",
+  "block.expandeddelight.cinnamon_pressure_plate": "肉桂压力板",
+  "block.expandeddelight.cinnamon_button": "肉桂按钮",
+  "block.expandeddelight.cinnamon_sign": "肉桂告示牌",
+  "block.expandeddelight.cinnamon_ceiling_hanging_sign": "肉桂悬挂告示牌",
+  "block.expandeddelight.cinnamon_cabinet": "肉桂橱柜",
+
+  "block.expandeddelight.asparagus_crate": "芦笋箱",
+  "block.expandeddelight.wild_asparagus": "野生芦笋",
+  "block.expandeddelight.asparagus_crop": "芦笋",
+
+  "block.expandeddelight.sweet_potato_crate": "红薯箱",
+  "block.expandeddelight.wild_sweet_potato": "野生红薯",
+  "block.expandeddelight.sweet_potato_crop": "红薯",
+
+  "block.expandeddelight.chili_pepper_crate": "辣椒箱",
+  "block.expandeddelight.wild_chili_pepper": "野生辣椒",
+  "block.expandeddelight.chili_pepper_crop": "辣椒",
+
+  "block.expandeddelight.wild_peanuts": "野生花生",
+  "block.expandeddelight.peanut_crop": "花生",
+
+  "block.expandeddelight.cranberry_plant": "蔓越莓植株",
+  "block.expandeddelight.cranberry_bag": "蔓越莓袋",
+  "block.expandeddelight.cranberry_cobbler": "蔓越莓馅饼",
+
+  "block.expandeddelight.salt_ore": "盐矿石",
+  "block.expandeddelight.deepslate_salt_ore": "深层盐矿石",
+
+  "block.expandeddelight.cheese_wheel": "奶酪轮",
+  "block.expandeddelight.goat_cheese_wheel": "山羊奶酪轮",
+  "block.expandeddelight.honeyed_goat_cheese_tart": "蜂蜜山羊奶酪挞",
+
+  "block.expandeddelight.juicer": "榨汁机",
+  "block.expandeddelight.cask": "发酵桶",
+  "block.expandeddelight.milk_cask": "牛奶桶",
+  "block.expandeddelight.goat_milk_cask": "山羊奶桶",
+  "block.expandeddelight.cheese_cask": "奶酪桶",
+  "block.expandeddelight.goat_cheese_cask": "山羊奶酪桶",

```

```suggestion
  "block.expandeddelight.milk_cask": "装有奶桶的发酵桶",
  "block.expandeddelight.goat_milk_cask": "装有山羊奶的发酵桶",
  "block.expandeddelight.cheese_cask": "装有奶酪的发酵桶",
  "block.expandeddelight.goat_cheese_cask": "装有山羊奶酪的发酵桶",
```
这些是上一行的发酵桶里装了对应东西的方块名吗
如是，建议照此修改，原版的炼药锅也是类似的名称

---

的确，已修改